game/fbo.hh

```diff
@@ -16,7 +16,8 @@ class FBO {
 	}
 	/// Handle clean-up
 	~FBO() {
-		if (m_fbo) glDeleteFramebuffers(1, &m_fbo);
+            if (glIsFramebuffer(m_fbo) == GL_TRUE)
```

Comment:
This can be shortened to `if (glIsFrambuffer(m_fbo)` right?

Reply:
I don't think `GLBooleans` are implemented as actual bool so no. Unless you want to get ants? Because that's how you get ants.

Reply:
well Yes and no: hopefully, GL_TRUE is !=0 but who can really say it will always be the case?...

Reply:
Fair enough

Reply:
that said comparing to TRUE is usually a bad idea

Reply:
done